llvm/lib/Target/EraVM/EraVMISelLowering.cpp

```diff
@@ -1105,6 +1105,41 @@ SDValue EraVMTargetLowering::LowerSELECT(SDValue Op, SelectionDAG &DAG) const {
   return DAG.getNode(EraVMISD::SELECT_CC, DL, VTs, Ops);
 }
 
+/// Lower U{add|sub|mul}.with.overflow feeding into a branch
+/// into {ADD|SUB|MUL}! and jump.of or jump.lt.
+SDValue EraVMTargetLowering::LowerBRCOND(SDValue Op, SelectionDAG &DAG) const {
+  SDValue Chain = Op.getOperand(0);
+  SDValue Cond = Op.getOperand(1);
+  SDValue Dest = Op.getOperand(2);
+  SDLoc DL(Op);
+  SDValue MatchedUArithO = matchingOverflowArithmeticOperation(Cond);
+  if (!MatchedUArithO)
+    return SDValue();
+
+  // Similar with the approach in function LowerSELECT, we will end up with two
```

Comment:
Nit: I wouldn't reference another function. It's not needed to understand what is happening while comments imo to reduce number of necessary lookups needed for understanding.

Reply:
Updated now.